pallets/automation-time/src/weights.rs

```diff
@@ -221,13 +221,9 @@ impl<T: frame_system::Config> WeightInfo for SubstrateWeight<T> {
 	// Storage: ParachainInfo ParachainId (r:1 w:0)
 	// Storage: UnknownTokens ConcreteFungibleBalances (r:1 w:0)
 	// Storage: AssetRegistry LocationToAssetId (r:1 w:0)
```

Comment:
`AssetRegistry` breaks this benchmark but manually adding weights in pallet for now until genesis available.